### PR TITLE
feat: add github issue and pr templete

### PR DESCRIPTION
**Pull request by CoderTH**
At present, the warehouse does not have a more standard pr and issue templates, I have added a preliminary version

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+_The template below is mostly useful for bug reports and support questions. Feel free to remove anything which doesn't apply to you and add more information where it makes sense._
```
Comment (lengrongfu) [inline .github/issue_template.md:1]: Can you add that the current issue is classified into bug, doc, feature, etc?

**Comment by chaunceyjiang** [comment]
I believe we should first add some labels to this repo.

**Comment by CoderTH** [comment]
> I believe we should first add some labels to this repo.

I believe we should first add some labels to this repo.

@archlitchi cc

**Comment by archlitchi** [comment]
> > I believe we should first add some labels to this repo.
> 
> Totally agree. Labels enhance organization
> 
> @archlitchi cc

agree, how can i cooperate? is this PR ready to merge?

**Comment by chaunceyjiang** [comment]
image

**Comment by CoderTH** [comment]
Do we need a robot to add label automatically?
@chaunceyjiang @archlitchi

**Comment by CoderTH** [comment]
@archlitchi can you help to merge this pr

**Pull request merged**